_security/access-control/cross-cluster-search.md

```diff
@@ -167,6 +167,9 @@ curl -k -XPUT -H 'Content-Type: application/json' -u 'admin:<custom-admin-passwo
   }
 }'
 ```
+All of the curl requests can also be sent using OpenSearch Dashboards Dev tools
+{: .label .label-green }
```

Comment:
```suggestion
{: .tip }
```
@AntonEliatra - You'll want to use [Callouts](https://github.com/opensearch-project/documentation-website/blob/main/FORMATTING_GUIDE.md#callouts) for this type of info, not [Labels](https://github.com/opensearch-project/documentation-website/blob/main/FORMATTING_GUIDE.md#labels).

Reply:
Can we add another H2 above this section? Something like, "Configuring cross-cluster search from Dashboards"? (I would rethink the other headings, too.)

Reply:
@hdhalter I dont think we discussed this today, can you elaborated please

Reply:
There is only one screenshot just to demonstrate that all of the curl commands can be executed from dev tools, Do you mean to add more screenshots and expand on this?